src/builder.rs

```diff
@@ -173,6 +173,26 @@ impl<St: BuilderState> ConfigBuilder<St> {
             .insert(Expression::from_str(key.as_ref())?, value.into());
         Ok(self)
     }
+
+    /// Sets an override if value is set
```

Comment:
I would like you to rephrase this into `Sets an override if value is Some(_)`. please. :smile: